### ISSUE TITLE
Use pills instead of dropdown menu for mobile navigation

### ISSUE THREAD

**Issue by jacobobryant** [jacobobryant/platypub#13]
Pills are nice cause you don't have to do an extra tap to make them appear. And if the user is on a tiny screen, they'll wrap. And no need for custom js/hyperscript. Here's some code I just copied from The Sample (need to at least change `bg-dark` to something else):

![image](https://user-images.githubusercontent.com/3696602/174415354-f58fe961-5f5c-4c7e-a77b-6a683d48039b.png)

```clojure
(defn pills [{:keys [options active]}]
  [:.flex.mx-3.sm:mx-0.overflow-x-auto.whitespace-nowrap
   (for [{:keys [value label href]} options]
     [:a.mr-1.p-2
      {:class (if (= active value)
                "bg-dark rounded text-white"
                "link")
       :href href}
      label])])
```

**Comment by jeffp42ker** [comment]
How does this look?

![pill-animation](https://user-images.githubusercontent.com/4767299/175010308-c57405e5-f130-4c35-acd5-c53ba656c7be.gif)

SVG icon source: https://orioniconlibrary.com/icon/sun-and-moon-4452
  License No attribution required for personal use - Free SVG & PNG icon
  For commercial use, credit us ...

**Comment by jacobobryant** [comment]
Looks good to me!

**Comment by jacobobryant** [comment]
Closed by #24
